targets/TARGET_RENESAS/TARGET_RZ_A1H/us_ticker.c

```diff
@@ -62,6 +62,7 @@ void us_ticker_init(void) {
     // INTC settings
     InterruptHandlerRegister(US_TICKER_TIMER_IRQn, (void (*)(uint32_t))us_ticker_interrupt);
     GIC_SetPriority(US_TICKER_TIMER_IRQn, 5);
+    GIC_SetConfiguration(US_TICKER_TIMER_IRQn, 3);
```

Comment:
The commit msg:

> Change the processes that depend on GR-EPACH.
> - These files are based on the codes of CMSIS5/RTX5 for RZ/A1H incorporated in DS-5.
 > - I modified the support range from Mbed OS 2 to Mbed OS 2 and 5.
 > - In "mbed_rtx.h", I changed the values depend on GR-PEACH like a other targets for Cortex-M.

Is this change related to the first one `These files are based on the codes of CMSIS5/RTX5 for RZ/A1H incorporated in DS-5. ` ? What does this change do? Changes like this should be in a separate commit - what is fixing and how.

Reply:
By the codes of CMSIS5/RTX5 for RZ/A1H incorporated in DS-5, `GIC_SetConfiguration` is a newly added function as Cortex-A, this function sets the interrupt configration using GIC.
It seems that this processing has become necessary for interrupt of each driver to work properly.

Reply:
Thanks, that answers it. Please consider this type of details to be part of the change (using git commit message).

You can amend the commit msg to include there this detail for the reader, it would answer my question.

Reply:
I amended the commit msg. Make a sense?